<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_tempco">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Temperature Coefficient</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_operating_temp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_resistance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_resistance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Resistance</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_operating_temp">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operating Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Tolerance</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmb_units">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tempco">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_power">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_power">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_watts">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
